backend/app/usecase/shortlink/metatag.go

```diff
@@ -0,0 +1,71 @@
+package shortlink
+
+import (
+	"github.com/short-d/short/backend/app/entity/metatag"
+	"github.com/short-d/short/backend/app/usecase/repository"
+)
+
+var _ MetaTag = (*MetaTagPersist)(nil)
+
+type MetaTag interface {
+	GetOGTags(alias string) (metatag.OpenGraph, error)
+	GetTwitterTags(alias string) (metatag.Twitter, error)
+}
+
+type MetaTagPersist struct {
+	shortLinkRepo repository.ShortLink
+}
+
+func (m MetaTagPersist) GetOGTags(alias string) (metatag.OpenGraph, error) {
```

Comment:
```suggestion
func (m MetaTagPersist) GetOpenGraphTags(alias string) (metatag.OpenGraph, error) {
```
In my opinion better to have a longer name but understandable than shorter